assets/common/js/models/GameController.js

```diff
@@ -58,8 +58,8 @@ GameController.prototype.enterWord = function() {
 	let wordToType = this.words.find(word => word.highlightInd > 0) // Find the word that is being typed (is highlighted; highlightInd > 0)
```

Comment:
There is a very rare and unlikely case in which this can fail. In the attached image, if I were to get rid of "constant" instead of "constantly", it wouldn't work because the find function will find the first highlighted word, which in this case is "constantly". Again, this is something that will never happen, since it requires a very specific condition.
1. There are two words with one of them containing the other word as a prefix substring. The containing word is positioned lower than the substring word.
2. Player wants to rid of the substring word first.
![Capture](https://user-images.githubusercontent.com/16367862/150658696-4bcdf4dd-7a43-4c0f-8c22-fd21816f7f0c.JPG)